CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Version 2.0.0
 
+- [#492]
+  - **Description:** Add autofocus directive on KRadioButton to fix autofocus behavior on dynamic rendering.
+  - **Products impact:** Kolibri's Setup Wizard
```

Comment:
Could you please change this to "bugfix"? This is not as much about the place it fixes, but rather a general category. There are some examples in the PR template you can choose from.